recharts-example/src/components/Profile.tsx

```diff
@@ -0,0 +1,24 @@
+import { User } from '../interface';
+
+type ProfileProps = {
+  user: User;
+};
+
+export default function Profile(props: ProfileProps) {
+  const { user } = props;
```

Comment:
if you don't pass user anywhere else, you could use even more destructuring here